tests/simulate/test_lammps.py

```diff
@@ -40,27 +45,50 @@ def ens_pot(self):
     def create_file(self):
         file_ = self.directory.file('test.data')
         with open(file_,'w') as f:
-            f.write(('LAMMPS test data\n'
-                     '\n'
-                     '5 atoms\n'
-                     '2 atom types\n'
-                     '\n'
-                     '-5.0 5.0 xlo xhi\n'
-                     '-5.0 5.0 ylo yhi\n'
-                     '-5.0 5.0 zlo zhi\n'
-                     '\n'
-                     'Atoms\n'
-                     '\n'
-                     '1 1 -4.0 -4.0 -4.0\n'
-                     '2 1 -2.0 -2.0 -2.0\n'
-                     '3 2 0.0 0.0 0.0\n'
-                     '4 2 2.0 2.0 2.0\n'
-                     '5 2 4.0 4.0 4.0\n'
-                     '\n'
-                     'Masses\n'
-                     '\n'
-                     '1 0.3\n'
-                     '2 0.1'))
+            if self.dim == 2: 
```

Comment:
`if-elif-else`

Could you maybe also reduced duplication here with string formatting? Something like what's above, since most of this is the same on both branches.